### PR TITLE
fix: use package version for app version in openapi

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
app version in openapi spec is hardcoded and outdated
This should be taken dynamically from the package version using the existing "get_eda_version" helper. 
Jira: https://issues.redhat.com/browse/AAP-37834

### CODE REVIEW

File: src/aap_eda/settings/default.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def _get_secret_key() -> str:
 def _get_boolean(name: str, default=False) -> bool:
     value = settings.get(name, default)
     if isinstance(value, str):
-        value = str_to_bool(value)
+        value = utils.str_to_bool(value)
```
Comment (mkanoor) [inline src/aap_eda/settings/default.py:142]: @Alex-Izquierdo Is this related to this PR?

Comment (Alex-Izquierdo) [reply src/aap_eda/settings/default.py:142]: I just import utils package for readability and usability, instead of importing functions one by one.

**Review by Dostonbek1 — APPROVED** [review]
